consensus_pbft.go

```diff
@@ -387,19 +393,21 @@ func (p *Pbft) runValidateState(ctx context.Context) { // start new round
 	}
 
 	quorum := p.state.getQuorumSize()
+	i := 1
 	for p.getState() == ValidateState {
-		_, span := p.tracer.Start(ctx, "ValidateState")
+		_, loopSpan := p.tracer.Start(ctx, fmt.Sprintf("ValidateState_Read#%d", i))
```

Comment:
To me, this counter should be metadata of the trace, not part of the name itself.

Reply:
Is it ok to add attribute to the span this way or you have different idea?
`span.SetAttributes(attribute.Int("loop.counter", counter))`

I mean maybe we don't even need this `counter` at all. Intention was just to differ span within for loop with this outer span for handling specific state.